data_diff/databases/base.py

```diff
@@ -392,10 +402,18 @@ def render_checksum(self, c: Compiler, elem: Checksum) -> str:
         return f"sum({md5})"
 
     def render_concat(self, c: Compiler, elem: Concat) -> str:
+        if self._prevent_overflow_when_concat:
+            items = [
+                f"{self.compile(c, Code(self.to_md5(self.to_string(self.compile(c, expr)))))}" for expr in elem.exprs
```

Comment:
render_concat/Concat in general isn't specific to hashing, so not sure this is the best place to add `self.to_md5`

Reply:
To me, this method is pretty generic, so I just have some concerns about introducing logic specific to hashing here

Reply:
I tried to find another way but looks like the current implementation is the right place because this part is common for all databases, and only here it is possible to prevent overflow.